solr/core/src/test-files/log4j.properties

```diff
@@ -7,6 +7,7 @@ log4j.appender.CONSOLE.layout=org.apache.log4j.EnhancedPatternLayout
 log4j.appender.CONSOLE.layout.ConversionPattern=%-4r %-5p (%t) [%X{node_name} %X{collection} %X{shard} %X{replica} %X{core}] %c{1.} %m%n
 log4j.logger.org.apache.zookeeper=WARN
 log4j.logger.org.apache.hadoop=WARN
+log4j.logger.org.apache.hadoop.security.token.delegation=DEBUG
```

Comment:
Rather than editing log4j.properties (which will affect the entire test suite) you can use an ```@LogLevel``` annotation on the test itself